chromium_src/chrome/app/chrome_main_delegate.cc

```diff
@@ -0,0 +1,7 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// This chormium_src override allows us to skip a lot of patching to chrome/BUILD.gn
+#include "brave/app/brave_main_delegate.cc"
+#include "../../../../chrome/app/chrome_main_delegate.cc"
```

Comment:
I decided I was tired of rebasing that BUILD.gn file for all the `brave_main_delegate.cc` references.

Reply:
When removing them(`brave_main_delegate.[cc|h]` from `chrome/BUILD.gn`, modifying them doesn't trigger rebuild.
If those files aren't changed frequently, how about adding some comment to those files?

Reply:
Or how about them to proper `BUILD.gn` in `brave` folder?

Reply:
I think we should solve this in a general way of touching the changed files.   Problem is there is a reference of the file in 10 different places otherwise.